.github/workflows/deploy-review.yml

```
@@ -1,50 +1,81 @@
+          name: ${{ env.FLY_APP_NAME }}
           secrets: GITHUB_TOKEN=${{ secrets.OCTOKIT_RESUME_TOKEN }}
+          image: ${{ env.IMAGE_NAME }}:pr-${{ github.event.number }}-${{ github.sha }}
 
       - name: 🗑 Delete deployment environment
         uses: strumwolf/delete-deployment-environment@v3
         if: ${{ github.event.action == 'closed' }}
         with:
           # ⚠️ The provided token needs permission for admin write:org
           token: ${{ secrets.GITHUB_TOKEN }}
-          environment: suddenlygiovanni-dev-${{ github.event.number }}
+          environment: ${{ FLY_APP_NAME }}
```

actionlint (1.7.4)

undefined variable "FLY_APP_NAME". available variables are "env", "github", "inputs", "job", "matrix", "needs", "runner", "secrets", "steps", "strategy", "vars"

(expression)